server/endpoints/Profile.test.ts

```diff
@@ -140,7 +176,7 @@ const mockVerificationTicket = jest.spyOn(Auth, 'getVerificationTicket')
   .mockImplementation(async (token?: string) => validTokenPayload);
 
 afterAll(async () => {
-  await mockVerificationTicket.mockRestore();
+  mockVerificationTicket.mockRestore();
```

Comment:
why did we remove `await` here?

Reply:
It looks like the method is returns void so we didnt need to have await there